conductor-clients/java/conductor-java-sdk/conductor-client/src/main/java/com/netflix/conductor/client/http/ConductorClient.java

```diff
@@ -442,6 +446,7 @@ public static class Builder<T extends Builder<T>>  {
         private long connectTimeout = -1;
         private long readTimeout = -1;
         private long writeTimeout = -1;
+        private long callTimeout = -1;
```

Comment:
@ystxn we need a way to set this private field in the builder

Reply:
my bad - thanks for spotting that